### ISSUE TITLE
Harvesting avocado halves and other morphology bugs.

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#32997]
(WEB REPORT BY: eneocho REMOTE: 206.221.180.138:7777)
# Revision
f839452895187cdf64c2996df53002bbcb35f050

# Description
While mutating morphology as a derelict mommi farmer I eventually found:
snack (yummy), one of the plants harvested into the placeholder inheritance snack item, still valid as food, but weird.
No icon fruit, works normally, but you gotta alt click or right click to pick it up
Avocado Half: one of the plants harvested into avocado halves, actual halves, not regular avocado.
Bugged icon fruit: when it pulls the "?" icon instead of whatever it intended.
# Steps to Reproduce
Mutate tons of morphology and wait to find one of these results.
# What you Expected
Regular crops coming from nonstandard crops
# What Actually Happened
the FUCK

**Comment by Eneocho** [comment]
![image](https://user-images.githubusercontent.com/67024428/182029714-8e9bba49-9d19-4ebe-b987-4da09ca3af06.png)

**Comment by Cartlord** [comment]
No-fruit can also mutate into (snack), and that can get frustrating.
